app/templates/view_teams.html

```diff
@@ -0,0 +1,95 @@
+{% import "toolkit/summary-table.html" as summary %}
+
+{% extends "_base_page.html" %}
+
+{% block page_title %}
+Buyers – Digital Marketplace admin
+{% endblock %}
+
+{% block breadcrumb %}
+{%
+      with items = [
+          {
+              "link": url_for('.index'),
+              "label": "Admin home"
+          },
+      ]
+  %}
+{% include "_breadcrumb.html" %}
+{% endwith %}
+{% endblock %}
+
+{% block main_content %}
+<div>
+  <h3>Team Details</h3>
+  Team ID: {{team_id}}
+  <br/>
+  Team Name: {{team.name}}
+</div>
+
+<div>
+  {% for i in team.teamLeads %}
+  <h3> Team Leads Information</h3>
+    {% for key, value in team.teamLeads[i].items() %}
```

Comment:
Check the indentation for this `for` block.  It looks a bit off to me.  Applies to another blocks below.